dev/doc/profiling.md

```diff
@@ -70,21 +76,42 @@ perf script --time '0%-100%'  |
 EOF
         ) |
         flamegraph.pl
-~~~~~
+```
 
-## Memory
+## Memory (memtrace)
+
+[memtrace](https://github.com/janestreet/memtrace) is a client library
+for OCaml's Memprof statistical memory profiler.
```

Comment:
You should mention somewhere that for this to work in Coq you must have memtrace installed when compiling Coq.

Reply:
Done thanks!